src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -61,13 +61,13 @@ <h2 class="card-pf-title">
                               outsideClick="true"
                               [popover]="missionContributeTemplate"
                               *ngSwitchCase="'not-implemented'">
-                          Not available <i class="pficon pficon-info"></i>
+                          Contribute now <i class="pficon pficon-info"></i>
```

Comment:
The text of these labels has been specified by UXD see https://github.com/fabric8-launcher/launcher-frontend/issues/341

Reply:
I will revert it and make a new issue to negotiate with the UXD team.